docs/source/customizing.rst

```diff
@@ -0,0 +1,90 @@
+
+Customizing nbconvert
+=====================
+
+Look at the first 20 lines of the ``python`` exporter
+
+.. code:: python
+
+    pyfile = !jupyter nbconvert --to python 'Index.ipynb' --stdout
+    for l in pyfile[20:40]:
+        print l
```

Comment:
:-P Yes, definitely. Probably whatever build system we have for the notebook docs will need to run the notebooks first, so that the output gets included when converting to rst.